### PR TITLE
LIT5839 40 41

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records

### CODE REVIEW

File: new/LIT5839ThirdEthioMaccabees.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2020-02-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
```
Comment (eu-genia) [inline new/LIT5839ThirdEthioMaccabees.xml]: add an abstract on the test and at least in relations clarify the relation between this and LIT1819 and LIT1820

File: new/LIT5840SecondEthioMaccabees.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2020-02-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
```
Comment (eu-genia) [inline new/LIT5840SecondEthioMaccabees.xml]: Same, you need an abstract and a set of relations to explain the relations between this and CAe 1819 and 1820

File: new/LIT5841EthioMaccabBooks.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5841EthioMaccabBooks" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Three Ethiopian Books of Maccabees</title>
```
Comment (eu-genia) [inline new/LIT5841EthioMaccabBooks.xml]: How is this different from 1819 and 1820?

File: new/LIT5839ThirdEthioMaccabees.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT5839ThirdEthioMaccabees.xml]: Done.

File: new/LIT5840SecondEthioMaccabees.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT5840SecondEthioMaccabees.xml]: Done.

File: new/LIT5841EthioMaccabBooks.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT5841EthioMaccabBooks.xml]: Done.

File: new/LIT5841EthioMaccabBooks.xml [hunk 4]
```diff
@@ -30,8 +30,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
-            </abstract>
+                <p>Three Ethiopian books of Maccabees are different from the Maccabees books
+                allegedly translated from Vulgata (<ref type="work" corresp="LIT1820Maccab"/>),
+                suchg as <ref type="work" corresp="LIT4302Maccab1"/>,
```
Comment (eu-genia) [inline new/LIT5841EthioMaccabBooks.xml]: such as

File: new/LIT5839ThirdEthioMaccabees.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT5839ThirdEthioMaccabees.xml]: I do not see new commits for this one

**Comment by eu-genia** [comment]
You may not merge your own commits. You should wait for me to approve and merge

**Comment by DenisNosnitsin1970** [comment]
ok.

File: new/LIT5841EthioMaccabBooks.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/LIT5841EthioMaccabBooks.xml]: Ok

**Pull request merged**